scormxblock/scormxblock.py

```diff
@@ -237,6 +237,7 @@ def resource_string(self, path):
         data = pkg_resources.resource_string(__name__, path)
         return data.decode("utf8")
 
+    @XBlock.supports('multi_device') # Mark as mobile-friendly
```

Comment:
Is this the only thing we need? @andrey-canon Does it need any CSS change?

Reply:
no @Squirrel18